tests/details/lending-bar-auto-renew.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test } from '../fixtures';
+
+/**
+ * auto-renew feature testing starts here.......
```

Comment:
SUGGESTION: It would be helpful to know what time duration in the real world this 5 minute duration is simulating.